includes/ampli/ampli-supported-platforms.md

```diff
@@ -0,0 +1,17 @@
+Ampli supports the following platforms - **Browser**, **Android**, **iOS**, **React Native**, **Node**, **Go**, **Python**, and **Java**.
+The other Amplitude SDKs for _Flutter_, _Unity_, and _Unreal_ _do not_ have Ampli at this time.
+
+??? note "Ampli Supported Platforms and SDKs"
+    | Platform     | Ampli Support | <div class="big-column">Supported Amplitude SDK(s)</div>                                                                                                                                                                                                                                                                                                                                                                                                  |
```

Comment:
Not sure how to make the table full width. Any suggestions?

Reply:
Can only make it full width with truncated lines between rows by simply adding to at the end of `docs/stylesheets/extra.css`

```.md-typeset__table {
  min-width: 100%;
}
```
<img width="656" alt="image" src="https://github.com/Amplitude-Developer-Docs/amplitude-dev-center/assets/31029607/30751b2e-1ee5-4b13-9c91-32ee34df54b5">
Not sure if this is better how we are now though